markdownlint/TOP005_blanksAroundMultilineHtmlTags/TOP005_blanksAroundMultilineHtmlTags.js

```diff
@@ -63,15 +63,20 @@ module.exports = {
         replacementText = `${replacementText}\n`;
       }
 
+      const ERROR_MESSAGE_BASE = "Expected a newline (\\n) or a code block delimiter (```)";
```

Comment:
"Blank line" is clearer than "newline (\n)" since people might actually try to put "\n" on the line before/after.
```suggestion
      const ERROR_MESSAGE_BASE = "Expected a blank line or a code block delimiter (```)";
```

Reply:
Ok, that's true!